cmd/translate/main_grpc_integration_test.go

```diff
@@ -470,3 +484,55 @@ func Test_ListMessages_gRPC(t *testing.T) {
 		})
 	}
 }
+
+func Test_UpdateMessages_gRPC(t *testing.T) {
+	t.Parallel()
+
+	ctx, subtest := testutil.Trace(t)
+
+	// Prepare
+	service := createService(ctx, t)
+
+	for i := 0; i < gofakeit.IntRange(1, 5); i++ {
+		uploadRequest := randUploadRequest(t, service.Id)
+		_, err := client.UploadTranslationFile(ctx, uploadRequest)
+		require.NoError(t, err, "create test translation file")
+	}
+
+	// Requests
+
+	tests := []struct {
+		request      *translatev1.UpdateMessagesRequest
+		name         string
+		expectedCode codes.Code
+	}{
+		{
+			name:         "Happy path, get all messages",
```

Comment:
'get all messages' might be misleading -> 'update successful' might be more appropriate.